test/e2e/e2e_test.go

```diff
@@ -2217,6 +2217,105 @@ func (t *PorchSuite) TestPodEvaluatorWithFailure(ctx context.Context) {
 	}
 }
 
+func (t *PorchSuite) TestPodEvaluatorWithLargeObjects(ctx context.Context) {
```

Comment:
Should this be merged with the TestLargePackageRevision test case

Reply:
Well, I believe this new testcase already in its current form incorporates what is tested in `TestLargePackageRevision`, making the later redundant. (Not a surprise, since some of the code was copy-pasted from there.) I didn't want to delete any tests, but if others agree that I am right, then we can delete TestLargePackageRevision.

Reply:
If we leave only one testcase, then we can rename the new test to `TestLargePackageRevision` if it explains the goal of the test better

Reply:
Either way is fine with me.

Reply:
I merged the two test cases: the new test case will always test updating a PRR with large amount of data, and skip only the Pod Evaluator test if tests are running without a function-runner